install_config/router/f5_router.adoc

```diff
@@ -26,6 +26,12 @@ like the
 xref:../../install_config/router/default_haproxy_router.adoc#install-config-router-default-haproxy[default
 HAProxy router].
 
+[IMPORTANT]
+====
+Support relationships between F5 and Red Hat provide a full scope of support for
+F5 integration, while Red Hat alone provides limited support for F5 integration.
+====
```

Comment:
I agree, the original wording here seems like it might cause more confusion than clarification. 

The main thing we need to highlight with a notice/warning like this is that F5 is managing the code/source for this integration, and that Red Hat is supporting the use of this through our support partnerships with F5 Networks. This means that while we (Red Hat) may help/interact in the troubleshooting process; bugs, feature enhancements, etc all get communicated to F5 Networks where they are managed/scheduled as part of their development/release cycles.